common/arm.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _ARM_H
+#define _ARM_H
+
+#ifndef __ASSEMBLY__
+
+#if __ARM_ARM < 6
+static inline void wfi(void)
+{
+	asm volatile (
+		"mov r0, #0\n\t"
+		"mcr p15, 0, r0, c7, c0, 4\n\t");
+}
```

Comment:
This seems unused right now but just in case it should list r0, cc and memory as clobbered

Reply:
"cc" should not be needed here. It's for when the code modifies flags in the CPSR directly or indirectly. "memory" should be there to prevent reordering by the compiler.